scripts/zones/Norg/npcs/Achika.lua

```diff
@@ -15,12 +15,8 @@ entity.onTrade = function(player, npc, trade)
 end
 
 entity.onTrigger = function(player, npc)
-    if player:hasKeyItem(xi.ki.TENSHODO_MEMBERS_CARD) then
-        if player:sendGuild(60421, 9, 23, 7) then
-            player:showText(npc, ID.text.ACHIKA_SHOP_DIALOG)
-        end
-    else
-        -- player:startEvent(150)
+    if player:sendGuild(60421, 9, 23, 7) then
```

Comment:
Can you buy from the Tenshodo NPCs without a membership?

Reply:
Yes, I made a new character on retail on purpose to test this. Was able to buy from all Norg vendors without tenshodo memebership.